.github/workflows/upload_release_asset.yml

```diff
@@ -0,0 +1,39 @@
+# fossa report attribution --debug --format spdx >> myFile
+name: Upload SBOM as Release Asset
+# Ensures that this is only run once, either when pushing
+# directly to main OR when pushing to a PR against main.
```

Comment:
Since we don't have a `main` branch for this repo, can you update this doc to indicate `master` or `develop` as appropriate?

Reply:
By the looks of things, this wouldn't happen on push. It'll run whenever the [release](https://github.com/cypress-io/cypress/releases) is created in Github (by Zenhub, in our case).

Reply:
Apologies for the confusion, I should have removed those comments.  They don't apply at all and were vestigial from another action I used.  [I have fixed this](https://github.com/cypress-io/cypress/pull/23097/commits/670a1a9ad73566bb1e8ab8426cdb2146ae357405), and also given @rockhold access to see the sample file in the sec.cypress repo.